lib/sendgrid_toolkit.rb

```diff
@@ -1,4 +1,5 @@
 require 'httparty'
+require 'pry'
```

Comment:
I think we shouldn't require `pry` here in production since it's only used in development.